css/media.css

```diff
@@ -0,0 +1,1039 @@
+  }
+
+  .products__product-card:nth-child(3) {
+    display: none;
+  }
+
+  .stories {
+    justify-content: space-evenly;
+  }
+
+  .stories-card:last-child {
+    display: none;
+  }
+
+  .gallery {
+    max-width: 80%;
+    margin: 0 auto;
+  }
+
+  .gallery__hashtag {
+    margin-right: 40px;
+  }
+
+  /* Отключение hover */
+  
+  .navbar__link:hover {
+    background-color: inherit;
+    color: inherit;
+  }
+
+  .link-more:hover {
+    background-color: inherit;
+  }
+
+  .footer-navbar__link:hover {
+    text-decoration: none;
+  }
+  
+  .footer-navbar__sign-up:hover {
+    color: inherit;
+    background-color: inherit;
+    opacity: 1;
+  }
+
+  .circles__link:hover {
+    transform: none;
+  }
+
+  .gallery__photo:hover {
+    filter: grayscale(100%);
+    transform: none;
+    background: none;
+    z-index: inherit;
+    padding: 0;
+  }
+  
+  .social__link:hover {
+    transform: none;
+  }
+}
+
+@media screen and (max-width: 900px) {
+  .header {
+    justify-content: space-between;
+    align-items: center;
+    padding: 0 5px;
+  }
+
+  .header__logo {
+    display: none;
+  }
+
+  .header__logo-mobile {
+    display: inline-block;
+    margin-top: 23px;
+  }
+
+  .navbar {
+    display: none;
+  }
+
+  .burger {
+    padding: 0;
+    border: none;
+    cursor: pointer;
+    position: relative;
+    display: inline-block;
+    width: 61px;
+    height: 9px;
+    margin-top: 5px;
+    background-color: #000;
+  }
+
+  .burger::after {
+    content: '';
+    position: absolute;
+    left: 0;
+    margin-top: 12px;
+    display: inline-block;
+    width: 61px;
+    height: 9px;
+    background-color: #000;
+  }
+  
+  /* market-section */
+
+  .video-block {
+    min-height: 400px;
+  }
+
+  .video-block__pic {
+    width: 500px;
+  }
+
+  .video-block__text-wrapper {
+    margin-top: 20px;
+    max-width: 30%;
+  }
+}
+
+@media (max-width: 800px) {
+  .footer {
+    position: relative;
+    z-index: 0;
+    padding: 27px 0 0 0;
+    height: 305px;
+  }
+
+  .footer-navbar {
+    max-width: 100%;
+    padding: 0 52px;
+    justify-content: space-between;
+  }
+
+  .footer-navbar__column {
+    flex-basis: 20%;
+    min-width: 0;
+    max-width: 155px;
+    margin: 30px 0;
+  }
+
+  .footer-navbar__title {
+    margin-bottom: 6px;
+  }
+
+  .footer-navbar__column:first-child {
+    flex-basis: 13%;
+  }
+
+  .footer-navbar__column:last-child {
+    flex-basis: 22%;
+  }
+
+  .footer-navbar__column:nth-child(3) {
+    padding-left: 3px;
+  }
+
+  .footer-navbar__sale-offer {
+    display: inline-block;
+    width: 155px;
+  }
+
+  .footer-navbar__column--social>.footer-navbar__title {
+    margin-bottom: 2px;
+  }
+
+  .footer-navbar__sign-up {
+    margin: 10px 0 27px 0;
+    padding-top: 5px;
+    width: 144px;
+    min-height: 27px;
+  }
+
+  .social {
+    width: 117px;
+    align-items: center;
+  }
+
+  .copyright {
+    margin-top: 41px;
+  }
+}
+
+
+@media screen and (max-width: 768px) {  
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected newline after "{" of a multi-line block (block-opening-brace-newline-after)](https://app.codacy.com/gh/WebHeroSchool/Axion2/pullRequest?prid=5654145)